Sources/armory/logicnode/OnRender2DNode.hx

```diff
@@ -0,0 +1,28 @@
+package armory.logicnode;
+
+import armory.trait.physics.PhysicsWorld;
```

Comment:
I think PhysicsWorld is not needed here.

Reply:
Good catch, thanks :)